js/common/lib/env.ts

```diff
@@ -65,7 +65,8 @@ export interface Env {
 /**
  * Represent a set of flags as a global singleton.
  */
-export const env: Env = {
+export const env: EnvIml = {
```

Comment:
this should not work. this looks like creating a plain object and reinterpret it as EnvImp

1. if you want to create a new instance of class EnvIml, use `new EnvIml()`.
2. do not export the internal class implementation. use export const env: Env = `